### PR TITLE
Deprbranch

### PR DESCRIPTION
**Pull request by bfgard**
I moved around a lot of the files and folders so it will say I deleted a lot of files. They're still there, but are in different folders.

### CODE REVIEW

File: Python/btax/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def joke():
```
Comment (talumbau) [inline Python/btax/__init__.py]: some leftover debug code?

**Comment by talumbau** [comment]
Can you describe for me the process of how you add files to your git commit? It seems too aggressive. You don't need to add the `.pyc` files and anything inside the `*-egg-*` directory are not needed. You may want to handle this with additional entries in the `.gitignore`.

File: Python/btax/__init__.py (hunk 1 above)
Comment (bfgard) [inline Python/btax/__init__.py]: Yes, that's from the tutorial I was a following. Is there a way to edit code in a pull request?

**Comment by bfgard** [comment]
I've been adding everything that appears in red in the git status, although I should be more judicious about what I'm committing.

I'll undo my commit and add things manually.

**Comment by jdebacker** [comment]
@talumbau We have a gitignore here: https://github.com/open-source-economics/B-Tax/blob/master/.gitignore

But it doesn't seem to be working well (e.g. the pyc files in Ben's PR).  Do we need one in each folder of the repo?  Something we are doing wrong?

**Comment by talumbau** [comment]
The `.gitignore` looks right to me. I don't have an explanation for why they are not ignored.

**Comment by bfgard** [comment]
I think it may have occurred because I changed the name of the .gitignore, but I was able to change it back.

**Pull request closed without merging**